src/ui/application-window.vala

```diff
@@ -268,6 +294,34 @@ namespace Peek.Ui {
         }
       }
     }
+    
+    
+    
+
+    [GtkCallback]
+    private void on_gif_button_clicked (Button source) {
+    	recorder.output_format="gif";
+	    record_button.set_label("Record as GIF");
```

Comment:
The string must be marked as translatable by wrapping it in `_()`, e.g. `record_button.set_label(_("Record as GIF"))`. I'd probably also change this to a single base string with printf like

    _("Record as %s").printf (_("GIF"))